addons/web/static/src/core/utils/objects.js

```diff
@@ -22,6 +22,22 @@ export function deepCopy(obj) {
     return JSON.parse(JSON.stringify(obj));
 }
 
+/**
+ * Returns a shallow copy of minuend with every property present in subtrahend
+ * removed.
+ *
+ * @param {Object} minuend object to copy
+ * @param {Object} subtrahend object with the properties to remove
+ * @return {Object}
+ */
+export function omit(minuend, subtrahend) {
+    return Object.fromEntries(
```

Comment:
personally, i prefer "stupid" code:
```js
const result = {};
for (let key in minuend) {
   if (!(key in subtrahend) {
    result[key] = minuend[key];
  }
}
return result;
```
Also, I would name the arguments something simpler, like obj1 and obj2.  Well, this is more a stylistic choice I guess